<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Введите число знаков после запятой : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="n_znak">
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_go">
        <property name="text">
         <string>Сосчитать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="log_field"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
